core/tests/test_utils.py

```diff
@@ -1019,13 +1021,26 @@ def set_collection_editors(self, collection_editor_usernames):
             self.set_user_role(name, feconf.ROLE_ID_COLLECTION_EDITOR)
 
     def get_user_id_from_email(self, email):
-        """Gets the user_id corresponding to the given email.
+        """Gets the user ID corresponding to the given email.
 
         Args:
             email: str. A valid email stored in the App Engine database.
 
         Returns:
-            user_id: str. ID of the user possessing the given email.
+            str. ID of the user possessing the given email.
+        """
+        gae_id = self.get_gae_id_from_email(email)
+        return (
+            user_services.get_user_settings_by_gae_id(gae_id).user_id)
+
+    def get_gae_id_from_email(self, email):
+        """Gets the GAE user ID corresponding to the given email.
+
+        Args:
+            email: str. A valid email stored in the App Engine database.
+
+        Returns:
+            str. GAE ID of the user possessing the given email.
         """
         return current_user_services.get_user_id_from_email(email)
```

Comment:
should this function be renamed for clarity? I think now code authors would be confused as to which "get_user_id_from_email" function should be used.

Reply:
Thanks for pointing this down. Done.